src/Tizen.NUI.Scene3D/src/public/Light/Light.cs

```diff
@@ -0,0 +1,127 @@
+    /// light.LookAt(new Vector3(1.0f, 1.0f, 1.0f));
+    /// sceneView.Add(light);
+    /// </code>
+    /// </example>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class Light : View
+    {
+        internal Light(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn)
+        {
+        }
+
+        /// <summary>
+        /// Create an initialized Light.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Light() : this(Interop.Light.LightNew(), true)
+        {
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        /// <summary>
+        /// Copy constructor.
+        /// </summary>
+        /// <param name="light">Source object to copy.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Light(Light light) : this(Interop.Light.NewLight(Light.getCPtr(light)), true)
+        {
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        /// <summary>
+        /// Maximum Activated Light Count.
+        /// It is possible to add more Lights to the SceneView than the Maximum Activated Light Count,
+        /// but only up to the Maximum Activated Light Count Lights will work.
+        /// </summary>
+        /// <remarks>
+        /// This property is read only.
+        /// </remarks>
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public uint MaximumActivatedLightCount
+        {
+            get
+            {
+                return Interop.Light.GetMaximumEnabledLightCount();
+            }
+        }
+
+        /// <summary>
+        /// The Light is turned on when the Light object is added on SceneView and enabled.
+        /// </summary>
+        /// <remarks>
+        /// SceneView can turn on only up to maximum enabled light count that can be retrieved by GetMaximumEnabledLightCount().
+        /// </remarks>
+        /// <param name="enable">True to make this Light enable.</param>
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void EnableLight(bool enable)
+        {
+            Interop.Light.Enable(SwigCPtr, enable);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        /// <summary>
+        /// Checks whether this light is enabled or not.
+        /// </summary>
+        /// <returns>
+        /// True if this light is enabled.
+        /// </returns>
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool IsLightEnabled()
```

Comment:
Just my curiosity :
Why do you not make Enabled property for LightEnabled, but separate them into the functions?
I think they can be
```
public bool Enabled
{ get; set; }
```

Reply:
Oh regarding this, @rabbitfor  already checked it ;)
Ignore my comment, please.

Reply:
I fixed it to Property